common/retinue_subunits/00_retinue_subunits.txt

```diff
@@ -0,0 +1,1746 @@
+  }
+}
+
+wc_ret_cul_twilhammer = {
+  first_type = 0
+  first_amount = 50
+
+  second_type = 6
+  second_amount = 350
+
+  special_troops = undead
+
+  potential = {
+    is_tribal = no
+    culture = twilights_hammer
+  }
+
+  modifier = {
+     light_infantry_offensive = 0.30
+     undead_offensive = 0.50
+  }
+}
+
+wc_ret_cul_dragonmaw = {
+  first_type = 2
+  first_amount = 125
+
+  second_type = 6
+  second_amount = 6
+
+  special_troops = dragons
+
+  potential = {
+    is_tribal = no
+    culture = dragonmaw
+  }
+
+  modifier = {
+     dragons_offensive = 0.10
+     pikemen_offensive = 0.30
+  }
+}
+
+wc_ret_cul_pygmy = {
+  first_type = 1
+  first_amount = 250
+
+  potential = {
+    is_tribal = no
+    culture =pygmy
+  }
+
+  modifier = {
+     heavy_infantry_defensive = 0.20
+     heavy_infantry_morale = 0.20
+  }
+}
+
+wc_ret_cul_quillboar = {
+  first_type = 1
+  first_amount = 250
+
+  potential = {
+    is_tribal = no
+    culture = quillboar
+  }
+
+  modifier = {
+     heavy_infantry_defensive = 0.30
+     heavy_infantry_morale = 0.10
+  }
+}
+
+wc_ret_cul_satyr = {
+  first_type = 1
+  first_amount = 150
+
+  second_type = 6
+  second_amount = 35
+
+  special_troops = archmage
+
+  potential = {
+    is_tribal = no
+    culture = satyr
+  }
+
+  modifier = {
+     heavy_infantry_offensive = 0.30
+     archmage_offensive = 0.30
+  }
+}
+
+wc_ret_cul_sethrak = {
+  first_type = 1
+  first_amount = 125
+
+  second_type = 3
+  second_amount = 125
+
+  potential = {
+    is_tribal = no
+    culture = sethrak
+  }
+
+  modifier = {
+     heavy_infantry_offensive = 0.30
+     light_cavalry_offensive = 0.20
+     light_cavalry_morale = 0.10
+  }
+}
+
+wc_ret_cul_tauren = {
+  first_type = 1
+  first_amount = 175
+
+  second_type = 4
+  second_amount = 75
+
+  potential = {
+    is_tribal = no
+    culture_group = tauren_group
+  }
+
+  modifier = {
+     heavy_infantry_defensive = 0.30
+     heavy_infantry_morale = 0.30
+     knights_defensive = 0.30
+     knights_morale = 0.30
+  }
+}
+
+wc_ret_cul_gurubashi = {
+  first_type = 3
+  first_amount = 200
+
+  potential = {
+    is_tribal = no
+    culture = gurubashi
+  }
+
+  modifier = {
+     light_cavalry_offensive = 0.20
+     light_cavalry_defensive = 0.40
+  }
+}
+
+wc_ret_cul_amani = {
+  first_type = 5
+  first_amount = 250
+
+  potential = {
+    is_tribal = no
+    culture = amani
+  }
+
+  modifier = {
+     archers_offensive = 0.30
+     archers_morale = 0.20
+  }
+}
+
+wc_ret_cul_zandalari = {
+  first_type = 2
+  first_amount = 125
+
+  second_type = 6
+  second_amount = 65
+
+  special_troops = large_cavalry
+
+  potential = {
+    is_tribal = no
+    culture = zandalari
+  }
+
+  modifier = {
+     large_cavalry_offensive = 0.30
+     large_cavalry_morale = 0.30
+     pikemen_morale = 0.50
+  }
+}
+
+wc_ret_cul_drakkari = {
+  first_type = 1
+  first_amount = 150
+
+  second_type = 3
+  second_amount = 75
+
+  potential = {
+    is_tribal = no
```

Comment:
In a game, a tribal monarchy is counted as a tribe, not a feudal.
Trolls and other tribal monarchies will not have a unique retinue until they become feudals.
But in Vanilla, you can hire a unique retinue, even if you are a tribe.